hail_scripts/v02/utils/computed_fields/vep.py

```diff
@@ -176,7 +183,7 @@ def get_expr_for_vep_sorted_transcript_consequences_array(vep_root, include_codi
             ]
         )
 
-    omit_consequence_terms = hl.set(["upstream_gene_variant", "downstream_gene_variant"])
+    omit_consequence_terms = hl.set(omit_consequences) if len(omit_consequences) > 0 else hl.empty_set(hl.tstr)
```

Comment:
Nitpicking, but `if omit_consequences else hl.empty_set(hl.tstr)` would be slightly more idiomatic.